misc/python/materialize/feature_benchmark/scenarios/benchmark_main.py

```diff
@@ -2182,3 +2182,46 @@ def benchmark(self) -> MeasurementSource:
                 """
             )
         )
+
+
+class ReplicaExpiration(Scenario):
+    def init(self) -> list[Action]:
+        return [
+            TdAction(
+                """
+> CREATE TABLE events (
+    content TEXT,
+    event_ts TIMESTAMP
+  );
+> CREATE MATERIALIZED VIEW last_30_days AS
+  SELECT event_ts, content
+  FROM events
+  WHERE mz_now() <= event_ts + INTERVAL '30 days';
```

Comment:
This was a view and index beforehand, what's the benefit of using a materialized view instead?

Reply:
Since we cannot test for diff counts directly in MVs, this benchmark indirectly tests that half the diffs are dropped even for materialized views.

Reply:
Reverted